Core/src/Material/Interactions.cpp

```diff
@@ -163,7 +163,7 @@ float Acts::computeEnergyLossBethe(const MaterialSlab& slab, int /* unused */,
   // the required modification only change the prefactor which becomes
   // identical to the prefactor epsilon for the most probable value.
   const auto running =
-      0.5f * std::log(u / I) + 0.5f * std::log(wmax / I) - rq.beta2 - dhalf;
+      std::log(u / I) + std::log(wmax / I) - 2.0f * rq.beta2 - 2.0f * dhalf;
```

Comment:
looking at `Eq. 33.5` - are we missing a square for `I`? or is this already absorbed in `const auto I = slab.material().meanExcitationEnergy();`?

Reply:
squres is not missing
`std::log(u/I) + std::log(wmax / I) = std::log(u * wmax/ I^2)` is eqaul to the first term in the paranthese